suzieq/engines/pandas/address.py

```diff
@@ -37,14 +39,13 @@ def get(self, **kwargs) -> pd.DataFrame:
         addnl_fields = ['master']
         drop_cols = []
 
-        v4addr = []
-        v6addr = []
-        macaddr = []
-        try:
+        if len(prefix) > 0:
```

Comment:
You can do "if prefix", no need to check for length each time, its more pythonic